src/deadline/nuke_submitter/deadline_submitter_for_nuke.py

```diff
@@ -78,7 +79,70 @@ def _handle_step(step: dict):
         _handle_step(step)
 
 
-def _get_job_template(settings: RenderSubmitterUISettings) -> dict[str, Any]:
+def _add_gizmos_to_job_bundle(job_bundle_path: Path, nodes: list[nuke.Node]) -> Optional[Path]:
+    gizmo_paths = [Path(node.filename()) for node in nodes if isinstance(node, nuke.Gizmo)]
+    if not gizmo_paths:
+        return None
+    gizmo_dir = job_bundle_path / "gizmos"
+    gizmo_dir.mkdir(parents=True, exist_ok=True)
+    for gizmo_path in gizmo_paths:
+        shutil.copy(gizmo_path, gizmo_dir / gizmo_path.name)
+    return gizmo_dir
+
+
+def _add_gizmo_dir_to_job_template(job_template: dict[str, Any], relative_gizmo_dir: Path) -> None:
+    if "parameterDefinitions" not in job_template:
+        job_template["parameterDefinitions"] = []
+
+    job_template["parameterDefinitions"].append(
+        {
+            "name": "GizmoDir",
+            "type": "PATH",
+            "description": "The directory containing Nuke Gizmo files used by this job.",
+            "default": str(relative_gizmo_dir),
```

Comment:
Any reason to not hard-code the default to `./gizmos` and instead add the gizmopath to the settings which would then allow a user to provide a different value via the parameter_values to the job bundle? Would prevent the need to modify and pass this directory around that's always going to be the same for this code